app/src/main/cpp/skyline/crypto/key_store.cpp

```diff
@@ -38,6 +38,14 @@ namespace skyline::crypto {
         titleKeys.emplace(key, valueArray);
     }
 
+    /**
+     * @note Any title keys which are already in the store will not have their values updated
+     */

```

Comment:
This comment would be in the header rather than the source file

Reply:
🤦‍♂️